cinder-{{cookiecutter.driver_name_lc}}/config.yaml

```diff
@@ -1,5 +1,37 @@
 options:
-  volume-dd-blocksize:
-    default: 1024
-    description: Disk block size for the volume
-    type: int
+{%- if cookiecutter.driver_install_method == 'PPA' %}
+  driver-source:
+    type: string
+    default:
+    description: |
+      Optional configuration to support use of additional sources such as:
+        - ppa:myteam/ppa
+        - cloud:trusty-proposed/kilo
+        - http://my.archive.com/ubuntu main
+      The last option should be used in conjunction with the key configuration
+      option.
+  driver-key:
+    type: string
+    default:
+    description: |
+      Key ID to import to the apt keyring to support use with arbitary source
+      configuration from outside of Launchpad archives or PPA's.
+{%- endif %}
+  volume-backend-name:
+    type: string
+    default:
+    description: |
+      Volume backend name for the backend. The default value is the
+      application name in the Juju model.
+  use-multipath:
+    type: boolean
+    default: True
+    description: |
+      Whether to use a multipath connection for iSCSI or FC in Cinder
+      volume service. Enabling multipath for VMs is managed by the
+      "use-multipath" option in the nova-compute charm.
```

Comment:
Do we have a logic to enable `use_multipath_for_image_xfer` and `enforce_multipath_for_image_xfer` when this option is set?

Reply:
It makes sense to offload the work to charm authors for driver specific configurations. However, a few options can be common across all of the driver in reality so it makes sense to have the handling of `volume-backend-name` and `use-multipath` out of the box in the template IMO.